app/models/comment.rb

```diff
@@ -1,2 +1,8 @@
 class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :user_id
+  belongs_to :post
+
+  def comments_counter
+    post.update_column(:comments_counter, post.comments.count)
+  end
```

Comment:
- [ ] [Optional] It would be better to use increment! as increment! method doesn't want us to save it manually. why it is better you can see [here](https://stackoverflow.com/questions/11499110/increment-vs-1). 
`example only` 
```ruby
def update_comments_counter
  post.increment!(:comments_counter)
end
```